CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -32,6 +32,13 @@
  */
 class CRM_Contribute_BAO_ContributionRecur extends CRM_Contribute_DAO_ContributionRecur {
 
+  /**
```

Comment:
@MiyaNoctem can you do git config core.filemode=false to stop committing file permission changes?

(This commit looks like it would be fairly easy to merge by itself it it were in a separate pr - although maybe not the tpl part if it doesn't apply over core by itself)